src/main/java/uk/gov/hmcts/divorce/divorcecase/model/FinalOrder.java

```diff
@@ -80,7 +81,19 @@
         label = "Does the applicant want to apply for Final Order and ${labelContentFinaliseDivorceOrEndCivilPartnership}?",
         access = {DefaultAccess.class}
     )
-    private YesOrNo doesApplicantWantToApplyForFinalOrder;
```

Comment:
we will need migration in Retired Field class

Reply:
if that is the case, i can just leave it as applicant. what do you think is better as a Back-end expert?

Reply:
I think changes in your PR is better as we would be able to identify exactly it's applicant1 or app2.